app/src/main/java/org/wikipedia/views/ImageTitleDescriptionView.kt

```diff
@@ -0,0 +1,88 @@
+package org.wikipedia.views
+
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.Gravity.CENTER_HORIZONTAL
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import androidx.core.content.ContextCompat
+import kotlinx.android.synthetic.main.view_image_title_description.view.*
+import org.wikipedia.R
+import org.wikipedia.util.DimenUtil
+
+internal class ImageTitleDescriptionView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyle: Int = 0) : LinearLayout(context, attrs, defStyle) {
+
+    init {
+        View.inflate(context, R.layout.view_image_title_description, this)
+        layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+        circularProgressBar.setCurrentProgress(80.0)
+    }
+
+    fun setTitle(titleText: String) {
+        title.text = titleText
+    }
+
+    fun setTitleTextSize(textSize: Float) {
+        title.setTextSize(TypedValue.COMPLEX_UNIT_SP, textSize)
+    }
+
+    fun setCaseForTitle(isAllCaps: Boolean) {
+        title.isAllCaps = isAllCaps
+    }
```

Comment:
Looks like this method has been used for only one time(and it's in the `init{}` of `taskImageDetailView`. Would it be possible to make it as a default setting?

Reply:
This is the title text. The same view is used for stats too, where it is not in caps, so this will have to be dynamically set, at one of the two places.